src/main/java/com/fitwsarah/fitwsarah/appointmentsubdomain/presentationlayer/AppointmentResponseModel.java

```diff
@@ -7,7 +7,7 @@
 
 @Value
 @Builder
-@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor(access = AccessLevel.PUBLIC)
```

Comment:
Can you give me insight on why you made the access level of your AllArgsConstructor for your response mapper PUBLIC?

Reply:
Yes i made it public because i had an error in one of test classes.